app/core/nlu/spacy.py

```diff
@@ -9,6 +11,130 @@ def __init__(self, nlp):
         
         # Load NLP model
         self.nlp = nlp
+
+        # Load Clist
+        self.cList = {
```

Comment:
Have the list be in a separate file in the next PR

Reply:
> Have the list be in a separate file in the next PR

okay , noted.